vars/RedHat_9.yml

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: MIT
+---
+# Put internal variables here with Red Hat Enterprise Linux 8 specific values.
```

Comment:
Would you mind updating the comment as well, to say 9 instead of 8?

Reply:
I updated it.